client/data/help/use-help-search-query.ts

```diff
@@ -14,41 +14,75 @@ export interface SearchResult {
 	source?: string;
 }
 
+interface TailoredArticles {
+	post_ids: Array< number >;
+	blog_id: number;
+	locale: string;
+}
+
 interface APIFetchOptions {
 	global: boolean;
 	path: string;
 }
 
-const fetchArticlesAPI = ( search: string, locale: string, sectionName: string ) => {
-	const queryString = buildQueryString( {
-		query: search,
-		locale,
-		section: sectionName,
-	} );
+const fetchArticlesAPI = async (
+	search: string,
+	locale: string,
+	sectionName: string,
+	articles?: TailoredArticles
+): Promise< SearchResult[] > => {
+	let queryString;
+	let articlesResponse: SearchResult[] = [];
+	let searchResultResponse: SearchResult[] = [];
 
+	if ( articles ) {
+		const { post_ids, blog_id, locale } = articles;
```

Comment:
We don't need `locale` here, and we don't need to pass it to the API! I also commented on the diff 👍